low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -166,8 +202,8 @@ def validate(self):
         pass
 
     def process(self):
-        # TODO: Import relevant python script here for further execution.
-        pass
+        from files.opt.seagate.sspl.setup import sspl_post_install
+        sspl_post_install.SSPLPostInstallCmd(self.args).process()
```

Comment:
Can you replace this by below like you did in other class.process()

`Cmd._call_script(f"{self._script_dir}/{self.script}", self._args)`

Reply:
As we have converted some files to python. that file will accept arguments like old we as well as a new way (Calling class with arguments more pythonic way).
Here we are calling those files in a more pythonic way.